server/src/test/java/io/spine/server/BoundedContextTest.java

```diff
@@ -181,15 +195,54 @@ void propagateRepositoriesToStand() {
         assertThat(stand.getExposedTypes(), contains(repository.getEntityStateType()));
     }
 
-    @Test
-    @DisplayName("not allow two aggregate repositories with aggregates of same state")
-    void throwOnSameAggregateState() {
-        ProjectAggregateRepository repository = new ProjectAggregateRepository();
+    @SuppressWarnings("unchecked")
+    @ParameterizedTest
+    @MethodSource("sameStateRepositories")
+    @DisplayName("not allow two entity repositories with entities of same state")
+    void throwOnSameEntityState(Pair<Repository, Repository> sameStateRepositories) {
+        Repository repository = sameStateRepositories.getKey();
```

Comment:
Can we use `Repository<?, ?>` here instead? Raw types are never OK.
Also, please narrow down the `unchecked` warning suppression and add a comment for it.